doc/test_organization/managing_tests_dependencies.qbk

```diff
@@ -7,17 +7,31 @@
 
 [section:tests_dependencies Managing test dependencies]
 
-Dependencies among tests are managed by means of [link boost_test.tests_organization.decorators decorators]. 
+In general, you should make sure that test cases are independent on one another.
```

Comment:
I would rather put the emphasis on the order: users should not rely on any execution order, and tests execution can be shuffled.